packages/blocks/src/class-blocks.php

```diff
@@ -17,7 +17,59 @@
  * @since 9.0.0
  */
 class Blocks {
+	/**
+	 * Get CSS classes for a block.
+	 *
+	 * @since 9.0.0
+	 *
+	 * @param string $slug  Block slug.
+	 * @param array  $attr  Block attributes.
+	 * @param array  $extra Potential extra classes you may want to provide.
+	 *
+	 * @return string $classes List of CSS classes for a block.
+	 */
+	public static function classes( $slug = '', $attr, $extra = array() ) {
```

Comment:
> `$slug = '', $attr,`

The default on `$slug` seems pointless since there's no default on `$attr`. IIRC PHP 8 is going to start complaining about that, too.

Reply:
👍 Done in 1a5e9878f33ca7ae274061065d6c9302131370a9

And asked in Slack about that PHP 8 change, I'm curious :)